qcp/QRegister.py

```diff
@@ -0,0 +1,43 @@
+from QMatrices import QMatrices
+import consts
+
+
+class QRegister:
+    def __init__(self, state):
+        """
+        Params
+        ------
+        State: List
+            Binary representation of the state of the QRegister
+        """
+
+        self.initState = state
```

Comment:
snake_case not pascalCase...

Reply:
Sorry, I couldn't find where pascalCase is written here? Could you kindly point out which line is it?

Reply:
"pascalCase" is the name given for the naming scheme of "initState" (first lowercase, next words capitalised), while in python the naming scheme generally used is called "snake_case", so `initState` -> `init_state`